### ISSUE TITLE
CSV export

### ISSUE THREAD

**Issue by ChristianMoesl** [benchie-io/benchie#38]
Implement an additional flag for `benchie show`, which renders the table as a CSV to the stdout. If this option is enabled, printing of warnings or other info messages should be prevented.

**Comment by ChristianMoesl** [comment]
@mwidmoser What do you think about that? Or would it be more reasonable to implement a `benchie export` command?

**Comment by mwidmoser** [comment]
> Or would it be more reasonable to implement a `benchie export` command?

No, I think a flag is fine. Maybe rather like `--csv out.csv` and the output (with warnings) remains the same?
